nncf/experimental/torch/fx/nncf_graph_builder.py

```diff
@@ -27,12 +28,43 @@ class GraphConverter:
     Builds the NNCFGraph from an torch.fx.GraphModule instance.
     """
 
+    def _get_node_subtype(
+        node: torch.fx.Node, metatype: om.OperatorMetatype, model: torch.fx.GraphModule
+    ) -> om.OperatorMetatype:
```

Comment:
This method looks common, but in fact, it works only with the **PTConv*dMetatype**. Why can't we unify this as was done for other backends? Or rename it like `is_depthwise`?